CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### [shopsys/framework]
 #### Changed
+- [#302 - All persistent files like uploads are now stored using abstract filesystem (Flysystem)](https://github.com/shopsys/shopsys/pull/302)
```

Comment:
Maybe name here what files are those files
please add here notice about using redis cache instead of file cache